gocourse03/micemetropolis/main.go

```diff
@@ -0,0 +1,157 @@
+package micemetropolis
+
+import (
+	"fmt"
+	"math/rand"
```

Comment:
```suggestion
	"math/rand/v2"
```

Ось тут почитати, чому треба використовувати v2 https://go.dev/blog/randv2